node/internal.ts

```diff
@@ -171,8 +171,27 @@ export function _setResourcePath(path: string): void {
 }
 
 /**
+ * Imports localized strings from json file.
+ * It can be used by common modules to load module specific loc strings
+ *
+ * @param     path      Full path to the json.
+ * @returns   void
+ */
+export function _importLocStrings(path: string): void {
```

Comment:
It _is_ overlaying the values in _locStringCache dictionary. I will add unit tests once we close on the approach.
Relaxing constraint in existing function _setResourcePath_ is also an option but will lead to confusion as we will really don't want to set more than one resource file. What we want is to import strings from more than one resx file. IMO, a better approach will be to add one more parameter called 'additionalResourceFiles' in setResourcePath function. Task will call it providing path to its task.json as well as array of all additional module.json files it has referenced from common folder
`tl.setResourcePath("path.join(__dirname, "task.json")", [path.join(__dirname, "node_modules", "azure-arm-rest", "module.json"), path.join(__dirname, "node_modules", "webdeployment-common", "module.json")])`